AutoGear.lua

```diff
@@ -1889,80 +1891,99 @@ AutoGearFrame:SetScript("OnEvent", function (this, event, arg1, arg2, arg3, arg4
 		elseif (event == "QUEST_DETAIL") then
 			QuestDetailAcceptButton_OnClick()
 		elseif (event == "GOSSIP_SHOW") then
-			--active quests
-			if (not IsClassic) then
-				local quests = C_GossipInfo.GetNumActiveQuests()
-				local info = {C_GossipInfo.GetActiveQuests()}
-				for i = 1, quests do
-					local title, questLevel, isTrivial, frequency, repeatable, isComplete, isLegendary = info[(i-1)*6+1], info[(i-1)*6+2], info[(i-1)*6+3], info[(i-1)*6+4], info[(i-1)*6+5], info[(i-1)*6+6], info[(i-1)*6+7]
-					if (isComplete) then
+			--active quests			
+			if (SL) then
```

Comment:
Please rename to ```IsSL``` to follow the convention.

Reply:
Also this needs to check ```IsBFA``` here, and not in the next ```if``` statement, as the next ```if``` statement is for Classic only.